retroshare-gui/src/gui/gxsforums/CreateGxsForumMsg.cpp

```diff
@@ -337,6 +340,20 @@ void  CreateGxsForumMsg::loadFormInformation()
 	//ui.forumMessage->setText("");
 }
 
+void CreateGxsForumMsg::checkLength()
+{
+	QString text;
+	RsHtml::optimizeHtml(ui.forumMessage, text);
+	std::wstring msg = text.toStdWString();
+	int charRemains = 199000 - msg.length(); // MAX_ALLOWED_GXS_MESSAGE_SIZE
```

Comment:
Do not hardcode the value, use the MAX_ALLOWED_GXS_MESSAGE_SIZE instead!

Reply:
yes, please update as hunbernd suggests.